src/Sentry/Internal/MainSentryEventProcessor.cs

```diff
@@ -148,9 +155,9 @@ public SentryEvent Process(SentryEvent @event)
                         Stacktrace = stackTrace
                     };
 
-                    @event.SentryThreads = @event.SentryThreads.Any()
+                    @event.SentryThreads = @event.SentryThreads?.Any() == true
                         ? new List<SentryThread>(@event.SentryThreads) { thread }
-                        : new[] { thread } as IEnumerable<SentryThread>;
+                        : new[] { thread }.AsEnumerable();
```

Comment:
Shorter -> no need to specify generic argument -> fewer chances to make a mistake.
Also `as` can theoretically return `null`, although not in this situation. It may happen for example if we change the left side to something else and forget to update the right side. `AsEnumerable()` avoids all those problems.

Reply:
I'm fine with changing this (best if we keep consistent with your approach going forward) even though it's more of a style thing. I personally can't move on from the fact it's a method call to simply make the compiler happy but one issue to me is that this is not really related to NRT so could be done on a separate PR, ideally.

Maybe a find and replace in the whole code base to make sure we stay consistent